services/migrations/gitea_uploader.go

```diff
@@ -801,6 +802,39 @@ func (g *GiteaLocalUploader) CreateReviews(reviews ...*base.Review) error {
 	return issues_model.InsertReviews(cms)
 }
 
+// UpdateTopics updates topics
+func (g *GiteaLocalUploader) UpdateTopics(topics ...string) error {
+	return g.CreateTopics(topics...)
```

Comment:
But maybe you need to delete the unused topics.

Reply:
Sure, I'll work on that